src/main/java/uk/gov/companieshouse/AppealApplication.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.companieshouse;
+
+import io.swagger.v3.oas.models.Components;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.Info;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
+public class AppealApplication {
+
+    public static void main (String[] args) {
+        SpringApplication.run(AppealApplication.class, args);
+    }
+
+    @Bean
+    public OpenAPI customOpenAPI() {
```

Comment:
Would it be possible to move it to separate bean factory, separate class? Having it here gives this class two responsibilities and at leas two reasons to change. If we move it to separate class we will have one class responsible for configuration of an open API component with single reason to change.

Reply:
Good point, this will be more readable 👍